demos/guide/quic-server-block.c

```diff
@@ -0,0 +1,322 @@
+/*
+ *  Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ *  Licensed under the Apache License 2.0 (the "License").  You may not use
+ *  this file except in compliance with the License.  You can obtain a copy
+ *  in the file LICENSE in the source distribution or at
+ *  https://www.openssl.org/source/license.html
+ */
+
+/*
+ * NB: Changes to this file should also be reflected in
+ * doc/man7/ossl-guide-quic-server-block.pod
+ */
+
+#include <string.h>
+
+/* Include the appropriate header file for SOCK_STREAM */
+#ifdef _WIN32 /* Windows */
+#include <stdarg.h>
```

Comment:
Missing space here:
```suggestion
# include <stdarg.h>
```
Same for all the other `#include` directives that are protected by `#if...else...endif`

Reply:
Do we do add the space like `# include` only when it's protected by if/else/endif? So the `#include` directives afterwards are fine?

Reply:
yes. the number of spaces should reflect the nesting of idefs
```
#include <stdio.h>
#ifdef LEVEL_1
# include <level_1.h>
# ifdef LEVEL_2
#  #include <level_2.h>
# endif
#endif
```
at least that's my understanding of [c-style](https://openssl-library.org/policies/technical/coding-style/)